src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/ExceptionHandling.cs

```diff
@@ -648,6 +648,11 @@ public static void RhThrowHwEx(uint exceptionCode, ref ExInfo exInfo)
         [StackTraceHidden]
         public static void RhThrowEx(object exceptionObj, ref ExInfo exInfo)
         {
+#if TARGET_WINDOWS
+            // Alert the debugger that we threw an exception.
+            InternalCalls.RhpFirstChanceException();
```

Comment:
```suggestion
            InternalCalls.RhpFirstChanceExceptionNotification();
```
More similarity with regular CoreCLR